api-epa/src/main/java/de/service/health/api/epa4all/EpaMultiService.java

```diff
@@ -144,6 +148,8 @@ private <T> T createProxyClient(Class<T> clazz, String backend, String serviceUr
         return clientFactory.createProxyClient(vauFacade, epaConfig.getEpaUserAgent(), clazz, getBackendUrl(backend, serviceUrl));
     }
 
+    // TODO - confirm purpose of the cache
```

Comment:
This make the lookup a lot faster. So the EPA of a patient should only be looked up a limited amount of time e.g. once.